applications/GeoMechanicsApplication/tests/cpp_tests/test_head_extrapolation_flow_workflow.cpp

```diff
@@ -10,112 +10,81 @@
 //  Main authors:    Jonathan Nuttall
 //
 
-#include <string>
 #include <iostream>
```

Comment:
I'm not sure why we would need to include `iostream` here. I do see that we use `std::filesystem`. I've tried to replace `iostream` by the more appropriate `filesystem` and that seems to work.
```suggestion
#include <filesystem>
```

What are your thoughts about this?

Reply:
Agreed, done!